common/gcloud.py

```diff
@@ -22,10 +22,10 @@
 from common import logs
 from common import new_process
 
-# Constants for dispatcher specs.
-DISPATCHER_MACHINE_TYPE = 'n1-standard-96'
-DISPATCHER_BOOT_DISK_SIZE = '4TB'
-DISPATCHER_BOOT_DISK_TYPE = 'pd-ssd'
+# TODO(metzman): Lower this when we figure out how else we can take advantage
+# of the 30 concurrent GCB builds.
```

Comment:
Dont understand this comment, can you explain more why we cannot go lower.

Reply:
Currently we use multiprocessing.Pool() which creates `N` processes for `N` cores.
GCB lets us do 30 concurrent builds.
I could always do 30 processes regardless of cores. I think this probably won't break because most processes will be waiting on GCB not doing expensive computation. I'm being cautious and not doing this for now since we've been bitten by parallelism in the past.

Reply:
Ok makes sense.
What will happen to other folks who will see default gcb limit to 10 ?

Reply:
I think it happens to work because GCB can queue up to 30.

Not a great answer but I haven't had time to make concurrent builds configurable. I thinkn I'll actually probably use the GCP API to find out instead.